### DISCUSSION TITLE
Welcome to ProjectNotes Discussions!

### DISCUSSION THREAD

**Discussion by kestermckinney** [kestermckinney/ProjectNotes#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LegoGuy32109** [comment]
yeah I can write comments on your discussions tab

**Reply by LegoGuy32109** [reply]
What I do is very surface level smoke tests for a arch linux user that has no PM experience

**Reply by kestermckinney** [reply]
Well you should switch to a more reliable distro like Fedora KDE.
